Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Android/CrashesDelegate.cs

```diff
@@ -1,145 +1,146 @@
-                return handler.Invoke(JavaObjectsConverter.ConvertErrorReport(report));
-            }
-            return true;
-        }
-
-        public bool shouldAwaitUserConfirmation()
-        {
-            var handler = shouldAwaitUserConfirmationHandler;
-            if (handler != null)
-            {
-                return handler.Invoke();
-            }
-            return false;
-        }
-
-        private AndroidJavaObject AttachmentWithText(string text, string fileName)
-        {
-            return _errorAttachmentLog.CallStatic<AndroidJavaObject>("attachmentWithText", text, fileName);
-        }
-
-        private AndroidJavaObject AttachmentWithBinary(byte[] text, string fileName, string contentType)
-        {
-            return _errorAttachmentLog.CallStatic<AndroidJavaObject>("attachmentWithBinary", text, fileName, contentType);
-        }
-
-        public AndroidJavaObject getErrorAttachments(AndroidJavaObject report)
-        {
-            if (GetErrorAttachments != null)
-            {
-                var logs = GetErrorAttachments(JavaObjectsConverter.ConvertErrorReport(report));
-                var nativeLogs = new List<AndroidJavaObject>();
-                foreach (var errorAttachmetLog in logs)
-                {
-                    AndroidJavaObject nativeLog = null;
-                    if (errorAttachmetLog.Type == ErrorAttachmentLog.AttachmentType.Text)
-                    {
-                        nativeLog = AttachmentWithText(errorAttachmetLog.Text, errorAttachmetLog.FileName);
-                    }
-                    else
-                    {
-                        nativeLog = AttachmentWithBinary(errorAttachmetLog.Data, errorAttachmetLog.FileName, errorAttachmetLog.ContentType);
-                    }
-                    nativeLogs.Add(nativeLog);
-                }
-
-                var javaList = new AndroidJavaObject("java.util.LinkedList");
-                if (nativeLogs.Count > 0)
-                {
-                    javaList.Call("addLast", nativeLogs[0]);
-                }
-                if (nativeLogs.Count > 1)
-                {
-                    javaList.Call("addLast", nativeLogs[1]);
-                }
-                return javaList;
-            }
-            else
-            {
-                return null;
-            }
-        }
-
-        public static void SetShouldAwaitUserConfirmationHandler(Crashes.UserConfirmationHandler handler)
-        {
-            shouldAwaitUserConfirmationHandler = handler;
-        }
-
-        public static void SetShouldProcessErrorReportHandler(Crashes.ShouldProcessErrorReportHandler handler)
-        {
-            shouldProcessErrorReportHandler = handler;
-        }
-
-        public static void SetGetErrorAttachmentsHandler(Crashes.GetErrorAttachmentsHandler handler)
-        {
-            GetErrorAttachments = handler;
-        }
-    }
-}
-#endif
+// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
The end-of-line change from `LF` to `CRLF` caused the large diff of this file.

Please change the end-of-line back to `LF` to avoid this.